src/UI/Implementation/Crawler/Entry/ComponentEntry.php

```diff
@@ -451,13 +451,25 @@ public function getExamplesPath(){
 		if(!$this->examples_path){
 			$path_componants = str_replace("Component","examples",$this->getPath())
 					."/".str_replace(" ", "", $this->getTitle());
-			$path_array = array_unique(explode ("/",$path_componants));
+			$path_array = self::array_iunique(explode ("/",$path_componants));
```

Comment:
This does not belong here. Nevertheless thank you. Please make a PR or commit directly, describing the need of this change in brief words.